utils/k8s/k8s.go

```diff
@@ -660,3 +660,26 @@ func (k8s K8S) BlockUntilPodIsUpOrTimeout(pod *core_v1.Pod, timeout time.Duratio
 
 	return k8s.BlockUntilPodIsUpWithContext(ctx, pod)
 }
+
+// CreatePersistentVolumeClaim is used to create the PVC in the given name space
+func (k8s K8S) CreatePersistentVolumeClaim(persistentVolumeClaim *core_v1.PersistentVolumeClaim, ns string) (result *core_v1.PersistentVolumeClaim, err error) {
```

Comment:
can you please change this function to something like this. 

```
func (k8s K8S) CreatePersistentVolumeClaim(persistentVolumeClaim *core_v1.PersistentVolumeClaim, namespace string) (*core_v1.PersistentVolumeClaim, error) {
	persistentVolumeClaimNameSpace := k8s.Clientset.CoreV1().PersistentVolumeClaims(namespace)
	return persistentVolumeClaimNameSpace.Create(persistentVolumeClaim)
}
```

Reply:
just curious why is this a good pattern?